crates/libcontainer/src/syscall/linux.rs

```diff
@@ -205,6 +205,29 @@ impl Syscall for LinuxSyscall {
         Some(user)
     }
 
+    fn get_pwdir(&self, uid: uid_t) -> Option<String> {
```

Comment:
How about using `nix::unixstd::Iser`? I think that It is better to use nix than to implement it ourselves.
https://docs.rs/nix/latest/nix/unistd/struct.User.html

Reply:
Oh, I'll check on it